homeassistant/components/samsungtv/media_player.py

```diff
@@ -164,7 +164,7 @@ def update(self) -> None:
 
         if self._attr_state == STATE_ON and self._app_list is None:
             self._app_list = {}  # Ensure that we don't update it twice in parallel
-            self.hass.async_add_job(self._update_app_list)
+            self.hass.add_job(self._update_app_list)
```

Comment:
What's the reason we want to schedule this call?

Reply:
I guess no reason - I had some delays on my old laptop but looks fine now so I've removed the schedule.